src/app/cart/cart-item/cart-item.component.sass

```diff
@@ -0,0 +1,22 @@
+.cart-item
+  &__info
+    display: inline-block
+    width: 300px
+    height: 24px
+    text-align: center
+
+  &__button
+    width: 24px
+    height: 24px
+    vertical-align: middle
+    padding: 0
+
+\:host
+  &.basic
+    color: green
+
+  &.highlighted
+    color: red
+
+    & div > span
```

Comment:
Brrr...

Reply:
Tag selector =(

Reply:
So what? What's wrong with tag selector?

Reply:
1. Not obvious
2. Potentially cause collisions in other places in the project
3. If you or someone else will change the tag then the rule will have to be changed also
4. To
5. Sjo
6. Pjatoje
7. Desjatoje

Reply:
`1. Not obvious`
Do not agree, it's pretty obvious to me.

`2. Potentially cause collisions in other places in the project`
Please  study how styles work in Angular 2+.
I see no possibility of collision, cause it has to be a `span` with a direct parent being a `div` inside a tag with `.highlighted` class, **in this particular component**.
It does not affect any other place in the project.

`3. If you or someone else will change the tag then the rule will have to be changed also`
Yep, if I ever decide to dramatically change the markup of the component, I will also have to change the styles.